Source/WebGPU/WGSL/tests/valid/overload.wgsl

```diff
@@ -202,3 +202,8 @@ fn testMatrixConstructor() {
     let m4 = mat4x4(vec4(0.0, 0.0, 0.0, 0.0), vec4(0.0, 0.0, 0.0, 0.0), vec4(0.0, 0.0, 0.0, 0.0), vec4(0.0, 0.0, 0.0, 0.0));
   }
 }
+
+fn testUnaryMinus() {
+  let x1 = -1;
```

Comment:
How does one write INT_MIN? If `-1` is interpreted as a unary minus followed by a positive number, there is no positive number that, when negated, results in INT_MIN

Reply:
That's a good question... I think the current implementation matches the spec, but we might need to handle it during parsing. I'll change the test to not use a constant, so it's still relevant either way.